packages/docs/components/tooltip.md

```diff
@@ -25,44 +25,54 @@ links:
 
 ## Anatomy
 
+<Anatomy img="/images/anatomy-tooltip.svg" />
+
+## Behavior
+
 <Description>
 
-<span class="is-fpo">Descriptive content around **tooltip anatomy** should go here.</span>
+Tooltips activate when a cursor hovers over an element, or an element receives focus.
+
+Tooltips can be triggered both by the shared parent container or the paired sibling. If you are describing an element that can have focus, no more work is necessary.
```

Comment:
> Tooltips can be triggered both by the shared parent container or the paired sibling. If you are describing an element that can have focus, no more work is necessary.

this feels hard to grok without a visual. Suggest either rewording or providing a visual 

cc: @joejezowski-okta

Reply:
This may be unnecessary here. Under the HTML tab, it'd be much easier to describe the setup.

Reply:
Reworded to feel more relevant to this section.